docs/swagger.yaml

```diff
@@ -110,6 +110,37 @@ info:
   title: Estuary API
   version: 0.0.0
 paths:
+  /admin/autoretrieve/init:
```

Comment:
Please add the proper annotations instead. This file will be automatically generated by our swagger workflow.

Reply:
I _think_ this was automatically generated when I did `make swagger`. Was I not supposed to do that?